roadservice/core/models.py

```diff
@@ -233,13 +293,93 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.type = Role.Type.COUNTY_MODERATOR
 
-    # TODO
-    def assign_county_expert(self):
-        pass
+    @property
+    def county(self):
+        return self.region.get_concrete()
 
     def get_concrete(self):
         return self
 
+    def add_service_team(self, speciality, team_members_users):
+        for team_member in team_members_users:
+            if team_member.has_role():
+                return None
+        new_team = ServiceTeam.objects.create(county=self.county, speciality=speciality)
+        for team_member in team_members_users:
+            Serviceman.objects.create(team=new_team, user=team_member)
+        return new_team
+
+    def edit_service_team(self, team, speciality, team_members_users):
+        ex_members_users = []
+        for member in team.members.all():
+            ex_members_users.append(member.user)
+        for user in team_members_users:
+            if user.has_role() and user not in ex_members_users:
+                return False
+        for user in team_members_users:
+            if user.has_role() and user in ex_members_users:
```

Comment:
Would you agree that only one of these two conditions would suffice?

Reply:
Personally, I prefer something like this, as I believe it'll be more readable:
```python
if user not in ex_members_users:
    Serviceman.objects.create(user=user, team=team)
```

Reply:
Yes, but it is more efficient to check if the user has a role first.